src/sentry/static/sentry/app/views/eventsV2/eventModalContent.tsx

```diff
@@ -34,57 +35,75 @@ type EventModalContentProps = {
  * Render the columns and navigation elements inside the event modal view.
  * Controlled by the EventDetails View.
  */
-const EventModalContent = (props: EventModalContentProps) => {
-  const {event, projectId, organization, location, eventView} = props;
-
-  // Having an aggregate field means we want to show pagination/graphs
-  const isGroupedView = hasAggregateField(eventView);
-  const eventJsonUrl = `/api/0/projects/${organization.slug}/${projectId}/events/${
-    event.eventID
-  }/json/`;
+class EventModalContent extends React.Component<EventModalContentProps> {
+  static propTypes = {
+    event: SentryTypes.Event.isRequired,
+    projectId: PropTypes.string.isRequired,
+    organization: SentryTypes.Organization.isRequired,
+    location: PropTypes.object.isRequired,
+  };
+
+  componentDidMount() {
+    const {event, organization} = this.props;
+    // metrics
+    trackAnalyticsEvent({
+      eventKey: 'discover_v2.event_details',
+      eventName: 'Discoverv2: Opened Event Details',
+      event_type: event.type,
+      organization_id: organization.id,
```

Comment:
The redash events are using `org_id` does that need to be the same here?

Reply:
@markstory I've been told that that's the case from @davidenwang https://github.com/getsentry/getsentry/blob/7ffbb15b4e33326f5fa5ea64f0f17ee5a9bb0c00/static/getsentry/gsApp/utils/trackAnalyticsEvent.jsx#L89